includes/database/engine/class-table.php

```diff
@@ -720,50 +836,31 @@ private function set_db_version( $version = '' ) {
 	}
 
 	/**
-	 * Get the table version from the database
+	 * Get the table version from the database.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 */
 	private function get_db_version() {
 		$this->db_version = $this->is_global()
 			? get_network_option( get_main_network_id(), $this->db_version_key, false )
 			:         get_option(                        $this->db_version_key, false );
-
-		/**
-		 * If the DB version is higher than the stated version and is 12 digits long, we need to update it to our
-		 * new, shorter format of 9 digits.
-		 *
-		 * This is only for 3.0 beta testers, and can be removed in 3.0.1 or above.
-		 *
-		 * @link https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7579
-		 */
-		if ( version_compare( $this->db_version, $this->version, '<=' ) || 12 !== strlen( $this->db_version ) ) {
-			return;
-		}
-
-		// Parse the new version number from the existing. Converting from {YYYY}{mm}{dd}{xxxx} to {YYYY}{mm}{dd}{x}
-		$date             = substr( $this->db_version, 0, 8 );
-		$increment        = substr( $this->db_version, 8, 4 );
-		$this->db_version = intval( $date . intval( $increment ) ); // Trims off the three prefixed zeros.
-
-		$this->set_db_version( $this->db_version );
 	}
 
 	/**
-	 * Delete the table version from the database
+	 * Delete the table version from the database.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 */
 	private function delete_db_version() {
 		$this->db_version = $this->is_global()
-			? delete_network_option( get_main_network_id(), $this->db_version_key )
-			:         delete_option(                        $this->db_version_key );
+			? delete_network_option( get_main_network_id(), $this->db_version_key, false )
+			:         delete_option(                        $this->db_version_key, false );
```

Comment:
```suggestion
			:         delete_option(                        $this->db_version_key );
```

Incorrect number of parameters.